W-18940999-merge-MS2025-06-ACB-to-latest-mk (#474)
mulesoft/docs-code-builder: modules/ROOT/pages/start-discover-ui.adoc

. Select the *Manage* (gear) icon from the activity bar.
. Select *Themes* > *Color Themes*, and select a theme, for example, *Dark Modern*:
+
image::acb-ui-dark-theme.png["IDE using the dark theme"]

== Explore the Canvas View

By default, when you open a project in Anypoint Code Builder, the canvas view opens first. This allows you to begin building and editing your flows visually, without switching to the XML editor.

> === Switch between views

Reviewer: Should this be title case?

Author: ```suggestion
=== Switch Views
```

You can use the toolbar to switch between the canvas and the code editor:
. Click the *</>* icon in the toolbar to toggle between canvas and code.
. To open both views side by side, right-click the canvas tab and select *Open in Canvas and Code Editor*.

NOTE: A dot (•) on a canvas tab indicates that the file contains unsaved changes.

> === Identify components with errors

Reviewer: Should this be title case?

Author: ```suggestion
=== Identify Components with Errors
```

Components that contain errors are marked with a red dot in the canvas.

> When you hover over the component, a tooltip appears listing the errors and warnings, including the field names involved.

Reviewer: ```suggestion
When you hover over the component, a tooltip lists the errors and warnings, including the field names involved.
```
